internal/interfaces/grpc/handler/operator.go

```diff
@@ -194,6 +194,81 @@ func (o operatorHandler) ListWebhooks(
 	return o.listWebhooks(ctx, req)
 }
 
+func (o operatorHandler) ListDeposits(
+	ctx context.Context,
+	req *pb.ListDepositsRequest,
+) (*pb.ListDepositsReply, error) {
+	return o.listDeposits(ctx, req)
+}
+
+func (o operatorHandler) ListWithdrawals(
+	ctx context.Context,
+	req *pb.ListWithdrawalsRequest,
+) (*pb.ListWithdrawalsReply, error) {
+	return o.listWithdrawals(ctx, req)
+}
+
+func (o operatorHandler) listDeposits(
+	ctx context.Context,
+	req *pb.ListDepositsRequest,
+) (*pb.ListDepositsReply, error) {
+	deposits, err := o.operatorSvc.ListDeposits(
+		ctx,
+		int(req.GetAccountIndex()),
+		domain.Page{
+			Number: int(req.GetPage().GetPageNumber()),
+			Size:   int(req.GetPage().GetPageSize()),
```

Comment:
This panics if the request does not contain any Page. Would be better to default to a fixed page in such case.

Reply:
Actually this wont panic since protobuf pkg has validation, i added default values.
Done